index.html

```diff
@@ -0,0 +1,289 @@
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have 
+                    a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social-platforms">
+                <h3>Lets Connect</h3>
+                <img src="images/social-media-icons.png" alt="social-media-icons" aria-hidden="true">
+              <ul>
+                  <li>
+                  <a href="https://twitter.com/RehemaMwaka1">
+                    <img
+                      src="./images/twitter.svg"
+                      class="icon"
+                      alt="twitter-icon"
+                    />
+                  </a>
+                </li>
+                 <li>
+                  <a href="https://www.linkedin.com/mwlite/in/rehema-mwaka-48a1801ab" class="icon">
+                      <img class="linkedini"
+                      src="./images/1.svg"
+                      alt="twitter-icon"
+                    />
+                    <img class="linkedin"
+                      src="./images/n.svg"
+                      alt="twitter-icon"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      src="./images/medium.svg"
+                      class="icon m"
+                      alt="medium-icon"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="https://github.com/TekyRey">
+                    <img
+                      src="./images/github.svg"
+                      class="icon"
+                      alt="github-icon"
+                    />
+                  </a>
+                </li>
+                
+                <li>
+                  <a href="#">
+                    <img
+                      src="./images/angel.svg"
+                      class="icon"
+                      alt="angellist-icon"
+                    />
+                  </a>
+                </li>
+              </ul>
+            </div>
+            
+                <button type="submit">Get my resume</button>
+            </section>
+        </div>
+    
+        <!-- Languages Section -->
+        <ul class="languages-section">
+            <li class="languages">
+                <div class="languages-title">
+                    <h2>Languages</h2>
+                    <img src="images/downarrow.svg" alt="">
+                </div>
+                
+                <ul class="language-list">
+                    <li class="language">
+                        <img src="images/js-icon.svg" alt="" aria-hidden="true">
+                        <p>JavaSript</p>
+                    </li>
+                    <li class="language">
+                        <img src="images/html-icon.svg" alt="" aria-hidden="true">
```

Comment:
I think it would be better if you add an `alt `message for your `languages `icon. This would help a user know the content of the `image` in case the image is not rendered on the browser.